p2p/src/net/default_backend/backend.rs

```diff
@@ -243,13 +242,13 @@ where
         announcement: Announcement,
     ) -> crate::Result<()> {
         let size = announcement.encode().len();
-        if size > ANNOUNCEMENT_MAX_SIZE {
+        if size > *self.p2p_config.max_message_size {
```

Comment:
I think this check is useless now. We already check the message size in `EncoderDecoder` and so this would always be `false`.